workflows/iuf/operations/managed-nodes/managed-nodes-rollout.yaml

```diff
@@ -169,10 +169,10 @@ spec:
           function main_sat_command {
               print_inputs_variables
               if [ -z "$managed_rollout_strategy" ] || [[ "$managed_rollout_strategy" == *"$JSONPATH"* ]] ; then
-                  echo "ERROR: No managed_rollout_strategy provided."
+                  echo "ERROR No managed_rollout_strategy provided. Cannot proceed further"
                   exit 1
               elif [ -z "$prepare_images_output" ] || [[ "$prepare_images_output" == *"$JSONPATH"* ]] ; then
-                  echo "ERROR: No prepare_images_output provided."
+                  echo "ERROR No output from 'prepare-images' stage. Cannot proceed further"
```

Comment:
You might add. "The managed-nodes-rollout stage relies on the images created in the prepare-images stage.  Run the prepare-images stage and try again"